packages/api-client/src/connections/SyncApi.ts

```diff
@@ -0,0 +1,44 @@
+import { sleep } from '@tupaia/utils';
```

Comment:
In Tamanu, we sync facility/mobile straight from central server. For datatrak, I was also debating between:
- Doing the same (ie: datatrak web connects with sync-server microservice)
- Or following the same Orchestrator - Microservice pattern (which means datatrak-web -> datatrak-server -> sync-server.

=> Decided to go with option because:
- It fits well with the existing architecture (including authorization, authentication etc)
- Connection between datatrak-server and sync-server is local, so it should be seamingless and have no bad impact on performance